src/main/kotlin/mrblrrd/fcmsend/FcmSend.kt

```diff
@@ -67,7 +68,7 @@ private class FcmSend : CliktCommand(
             ).apply { doOutput = true }
 
         DataOutputStream(connection.outputStream).use { outputStream ->
-            outputStream.writeBytes(message)
+            outputStream.write(message.encodeToByteArray())
```

Comment:
Thank you! I have the only question.

```suggestion
            outputStream.write(message.toByteArray(Charsets.UTF_8))
```
What's the reason for using experimental API instead of `toByteArray` which is used under the hood? Is this for multiplatform?

Reply:
Yeah, you're right, my mistake

Reply:
Not really a mistake. It’s a good idea to make this project MPP... 🤔 Thanks.